libraries/abstractions/common_io/test/test_iot_tsensor.c

```diff
@@ -39,6 +39,7 @@
 #include "FreeRTOS.h"
 #include "semphr.h"
 
+#define testIotTSENSOR_DELAY              pdMS_TO_TICKS( 5000 )
```

Comment:
In general when a `#define` macro or variable has units, add the units to the end of the name.  This helps reviewers and maintainers spot misuse.  Here we would prefer the name `testIoTSENSOR_DELAY_TICKS`.